### PR TITLE
Updated legion submodule

### PR DESCRIPTION
**Pull request by ipdemes**

### CODE REVIEW

**Comment by ipdemes** [comment]
@junghans : is it possible to see " /home/flecsi/tpl/build/legion-prefix/src/legion-stamp/legion-build-err.log" file from the failing builds in TRAVIS? I am trying to figure out what is wrong there...

File: docker/Dockerfile [hunk 1]
```diff
@@ -14,7 +14,8 @@ RUN mkdir -p build && cd build && \
     cmake -DBUILD_SHARED_LIBS=ON \
           -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE} \
           -DUSE_SYSTEM_LIBS=${SYSTEM_LIBS} \
-          .. && make -j2;
+          .. && make -j2 && \
+	  cat legion-prefix/src/legion-stamp/legion-build-err.log;
```
Comment (junghans) [inline docker/Dockerfile]: I will have to do `|| { cat ; exit 1;}` instead of `&& cat`.

**Comment by ipdemes** [comment]
@junghans , Ok, I have tried to correct Dockerfile, but it seems like I use some wrong syntax there. Could you, please, check this out?

**Comment by ipdemes** [comment]
@junghans : now tests fail because of "No space left on device" error. could you, please, clean-up the machine and rerun the tests?

**Comment by junghans** [comment]
There is nothing I can do, but re-trigger.

**Pull request merged**